lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -179,6 +179,23 @@
             - When a default security group is created for a Linux host a rule will be added allowing inbound TCP
               connections to the default SSH port 22, and for a Windows host rules will be added allowing inbound
               access to RDP ports 3389 and 5986. Override the default ports by providing a list of open ports.
+    enable_accelerated_networking:
+        description:
+            - This enables accelerated networking on the network interface
+        aliases:
+            - accelerated_networking
+        type: bool
+        default: 'no'
```

Comment:
false?

Reply:
Everywhere else I've seen a default given for a boolean parameter it has been `'no'`, e.g. the documentation variable for `ec2_vol`. Are those incorrect?